ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -359,3 +384,160 @@ def req_dut(self, cmd):
         reply = self.q_from_dut.get()
         self.log("reply: %s" % reply)
         return reply
+
+    # make sure setUp is done before run this test case
+    def test_wrarp_advance(self):
+        release = self.get_release_version()
+        if release != 'none' and release < '202205':
+            self.log("release version does not support advance test case")
+            return
+
+        for test in self.tests:
+            port = random.choice(test['acc_ports'])
+            test_non_broadcast_reply_thread = threading.Thread(target=self.test_non_broadcast_reply_thr, kwargs={'port': port})
```

Comment:
In the description, it says 3 testcases, but here, only one, can I understand 3 test objectives descripted are satisfied in this advance case?

Reply:
3 testcase actually mixed together in implementation, as we discussed I will split them into 2 cases to make them can be called separately.